### PR TITLE
Rhed form screen

### PR DESCRIPTION
**Pull request by Rhedyy**
Fixed The Bottom Modal Sheet, adjusted the colorways for the slider, made it more compact.

### CODE REVIEW

**Comment by Hjkun77** [comment]
Can you add in your Pull Request(PR)) a photo or video of your code? @Rhedyy

**Comment by Rhedyy** [comment]
code

**Comment by Hjkun77** [comment]
Oh i meant the itsura ng form not the code sir my bad @Rhedyy

**Comment by Rhedyy** [comment]
Screenshot_1596779146

File: lib/main.dart [hunk 1]
```diff
@@ -42,6 +42,23 @@ class _MainScreenState extends State<MainScreen> {
     });
```
Comment (Hjkun77) [inline lib/main.dart:42]: put all the things in main.dart in class MyApp

Comment (Hjkun77) [reply lib/main.dart:42]:
```
class MyApp extends StatelessWidget {
@override
  Widget build(BuildContext context) {	  Widget build(BuildContext context) {
    return Scaffold(	    return Scaffold(
@@ -64,6 +81,7 @@ class _MainScreenState extends State<MainScreen> {
        floatingActionButton: FloatingActionButton(	        floatingActionButton: FloatingActionButton(
          onPressed: () {	          onPressed: () {
            print("Floating Add Button Clicked");	            print("Floating Add Button Clicked");
            displayBottomSheet(context);
          },	          },
          tooltip: 'Add',	          tooltip: 'Add',
          child: Icon(Icons.add),	          child: Icon(Icons.add),

void displayBottomSheet(BuildContext context) {
    showModalBottomSheet(
        isScrollControlled: true,
        context: context,
        builder: (ctx) {
          return Container(
            height: MediaQuery.of(context).size.height * 0.885,
            child: Center(
              child: Padding(
                padding: const EdgeInsets.fromLTRB(40 , 40 , 40, 40),
                child: DataForm(),
              ),
            ),
          );
        });
  }
}

something like that

File: lib/screens/form.dart [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+  _DataFormState createState() => new _DataFormState();
+}
+
+class _DataFormState extends State<DataForm> {
+  final _formKey = GlobalKey<FormState>();
+
+  String _coursename = '';
+  String _code = '';
+  int _units = 0;
+  var myLetterGrade = "W";
```
Comment (Hjkun77) [inline lib/screens/form.dart:14]: make the variable consistent if

with underline or without

Comment (Hjkun77) [reply lib/screens/form.dart:14]: better to make this String na din that a `var`

File: lib/screens/form.dart [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+  var sliderValue = 0.0;
```
Comment (Hjkun77) [inline lib/screens/form.dart:15]: make this into an int or double instead

File: lib/screens/form.dart [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+                                myLetterGrade = "A";
+                                _grade = 4.0;
+                              }
+                            },
+                          );
+                        },
+                      ),
+                    ),
+                  ),
+                  Align(
```
Comment (Hjkun77) [inline lib/screens/form.dart:230]: No need to for align if you'll use container...container has a align attribute already

**Review by Hjkun77 — APPROVED** [review]
Please address these!

Thank you @Rhedyy you did a great job!

**Comment by Rhedyy** [comment]
Copy Sir! Will address these soon!

**Pull request merged**